app/models/well.rb

```diff
@@ -96,7 +96,7 @@ def attach(wells)
           )
         }
 
-  scope :without_report, ->(product_criteria) { where.not(id: Well.with_report(product_criteria)) }
+  scope :without_report, ->(product_criteria) { where.not(id: with_report(product_criteria)) }
 
```

Comment:
A bit like the change above, this is due to changes in the way in which rails handles scopes. https://github.com/rails/rails/issues/14003